Fix 3.34 test pipeline

**Describe the bug**
Test pipeline for 3.34 is [not working](https://github.com/Joonalai/profiler-qgis-plugin/actions/runs/13956066004/job/39067350812) at the moment.

**To Reproduce**
Add release_34 to qgis-image-tags matrix and push a commit.

**Expected behavior**
Tests should run properly.

**Traceback**

<details>
<summary>Traceback</summary>

```
 ==================================== ERRORS ====================================
_______________ ERROR collecting test/plugin/test_macro_model.py _______________
test/plugin/test_macro_model.py:22: in <module>
    from profiler_plugin.ui.macro.macro_model import MacroTableModel
/usr/lib/python3/dist-packages/qgis/utils.py:892: in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
.venv/lib/python3.10/site-packages/profiler_plugin/__init__.py:21: in <module>
    from profiler_plugin.plugin import ProfilerPlugin
/usr/lib/python3/dist-packages/qgis/utils.py:892: in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
.venv/lib/python3.10/site-packages/profiler_plugin/plugin.py:34: in <module>
    from profiler_plugin.ui.macro.macro_panel import MacroToolFactory
/usr/lib/python3/dist-packages/qgis/utils.py:892: in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
.venv/lib/python3.10/site-packages/profiler_plugin/ui/macro/macro_panel.py:38: in <module>
    UI_CLASS: QWidget = load_ui_from_file(
.venv/lib/python3.10/site-packages/qgis_plugin_tools/tools/resources.py:350: in load_ui_from_file
    ui_class, _ = uic.loadUiType(ui_file_path)
/usr/lib/python3/dist-packages/qgis/PyQt/uic/__init__.py:36: in __loadUiType
    return __PyQtLoadUiType(*args, **kwargs)
/usr/lib/python3/dist-packages/PyQt5/uic/__init__.py:200: in loadUiType
    winfo = compiler.UICompiler().compileUi(uifile, code_string, from_imports,
/usr/lib/python3/dist-packages/PyQt5/uic/Compiler/compiler.py:111: in compileUi
    w = self.parse(input_stream, resource_suffix)
/usr/lib/python3/dist-packages/PyQt5/uic/uiparser.py:1020: in parse
    document = parse(filename)
/usr/lib/python3.10/xml/etree/ElementTree.py:1222: in parse
    tree.parse(source, parser)
/usr/lib/python3.10/xml/etree/ElementTree.py:569: in parse
    source = open(source, "rb")
E   FileNotFoundError: [Errno 2] No such file or directory: '/__w/profiler-qgis-plugin/profiler-qgis-plugin/.venv/lib/python3.10/site-packages/profiler_plugin/ui/macro/macro_panel.ui'
```
</details>

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -28,6 +28,4 @@ jobs:
           .venv/bin/pip3 install -q -r requirements.txt --no-deps --only-binary=:all:
           .venv/bin/pip3 install . --no-deps
       - run: |
-          .venv/bin/pytest
-        env:
-          QT_QPA_PLATFORM: offscreen
+          .venv/bin/pytest --qgis_disable_gui

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -13,7 +13,7 @@ jobs:
       image: qgis/qgis:${{ matrix.qgis-image-tags }}
     strategy:
       matrix:
-        qgis-image-tags: ["3.40", latest]
+        qgis-image-tags: [release-3_34, "3.40", latest]
       fail-fast: false
 
     steps:
@@ -24,6 +24,7 @@ jobs:
       - run: |
           python3 -m venv --system-site-packages .venv
       - run: |
+          .venv/bin/pip3 install -q -U pip
           .venv/bin/pip3 install -q -r requirements.txt --no-deps --only-binary=:all:
           .venv/bin/pip3 install . --no-deps
       - run: |

